swahili-syntax/syntaxes/swahili.tmLanguage.json

```
@@ -0,0 +1,113 @@
+{
+	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+	"name": "swahili",
+	"patterns": [
+		{
+			"include": "#keywords"
+		},
+		{
+			"include": "#strings"
+		},
+		{
+			"include": "#numerical values"
+		},
+		{
+			"include": "#Functions"
+		},
+		{
+			"include": "#single line comments"
+		},
+		{
+			"include": "#Multi-line comments"
+		},
+		{
+			"include": "#Flow Control"
+		},
+		{
+			"include": "#Function Names"
+		},
+		{
+			"include": "#Identifier"
+		},
+		{
+			"include": "#Function Call"
+		}
+	],
+	"repository": {
+		"keywords": {
+			"patterns": [{
+				"name": "keyword.control.swahili",
+				"match": "\\b(wacha|shughuli|kweli|uwongo)\\b"
+			}]
+		},
+		"strings": {
+			"name": "string.quoted.double.swahili",
+			"begin": "\"",
+			"end": "\"",
+			"patterns": [
+				{
+					"name": "constant.character.swahili",
+					"match": "\\\\."
+				}
+			]
+		},
+		"numerical values":{
+			"patterns": [{
+				"name": "constant.numeric.swahili",
+				"match": "\\d+(.\\d+)?"
+			}]
+		},
+		"single line comments": {
+			"patterns": [{
+				"name": "comment.line.swahili",
+				"match": "\/\/.+"
+			}]
+		},
+		"Multi-line comments": {
+			"name": "comment.block.swahili",
+			"begin": "\/\\*",
+			"end": "\\*\/",
+			"patterns": [
+				{
+					"name": "comment.block.swahili",
+					"match": "\\\\."
+				}
+			]
+		},
+		"Functions": {
+			"patterns": [
+				{
+					"name": "entity.name.function.swahili",
+					"match": "\\b(shughuli|tupu|niNambari|soma|andika|somaNambari|futa|niShughuli|idadi|niNambari|niJina|niOrodha|Nambari|niTupu )\\b"
```

For one, these may be renamed, and we're continually adding more. There is no need to highlight built in function calls any different than function calls
```suggestion
					"match": "\\b(soma|andika|somaNambari|futa|niShughuli|idadi|niNambari|niJina|niOrodha|Nambari|niTupu|Jina)\\b"
```

---

Done